src/main/java/online/umbcraft/libraries/utils/MaterialUtil.java

```diff
@@ -0,0 +1,27 @@
+package online.umbcraft.libraries.utils;
+
+import online.umbcraft.libraries.GoldenDupes;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.inventory.ItemStack;
+
+public class MaterialUtil {
+
+    public static boolean isTotem(Material m) {
+        if (GoldenDupes.getInstance().serverVersion() <= 12) {
+            return m == Material.valueOf("TOTEM");
```

Comment:
I believe this code also may break when running on versions under 1.11, since totems were added in 1.11. Calling valueOf on an enum which doesn't exist will throw an IllegalArgumentException - this case can probably be solved either with another version check which simply returns false if the version is below 11, or (less ideally) by catching the exception as the previous code did